<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TakeSkipEditor</class>
 <widget class="QWidget" name="TakeSkipEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btnInfo">
       <property name="toolTip">
        <string extracomment="HELLO"/>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="accessibleDescription">
        <string extracomment="Description"/>
       </property>
       <property name="text">
        <string notr="true">?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_takeSkip">
       <property name="text">
        <string>Take Skip String</string>
       </property>
       <property name="buddy">
        <cstring>le_takeSkip</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_takeSkip">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Roboto Mono&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_deinterleaved">
       <property name="text">
        <string>Deinterleave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_deinterleave">
       <property name="suffix">
        <string> channels</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_interleaved">
       <property name="text">
        <string>Interleave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_frameBased">
       <property name="text">
        <string>Frame-based</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
